### ISSUE TITLE
Desktop window scroll/zoom behaviour

### ISSUE THREAD

**Issue by naxels** [Wisser/Jailer#86]
### Is your feature request related to a problem? Please describe.

Working in the Data Browser, on the Desktop tab on a macOS laptop and a Magic Mouse 2/Trackpad I often find
the tables/table windows & relationships zooming in/out while what I wanted was to navigate/scroll across the table data or the entire display instead of the zoom level. I find myself constantly readjusting the zoom level.

Data Browser -> Window menu:
<img width="255" alt="image" src="https://user-images.githubusercontent.com/3195443/154297854-567f0e2b-dc2e-43ca-81af-40af621ae7ca.png">

Default behaviour on a Mac is when you use the scroll-wheel, you move the page/document/window further along instead of adjusting the zoom level.

### Describe the solution you'd like

Create a way to distinguish between Navigating the Desktop tab and Zooming the Desktop tab, besides the little box on the left side of the window below Navigation.

Some of the ideas I've come up with:
- Add an option: Scrolling will zoom (default) Yes/No
- Add a keyboard shortcut + scrolling command that changes the behaviour (example on Mac: Command + Scroll = Zoom, while normal Scroll = Navigate the display)
- Since I rarely zoom in/out, and use the little window on the left to navigate more often: disable the scroll wheel for zooming, leave the keyboard shortcuts in case you want to

### Describe alternatives you've considered

For the past week+ I've been trying to adjust to the scroll = zoom behaviour when not scrolling within a table, but often OS habits take over and I find the entire zoom level adjusted (also Magic Mouse 2 is pretty sensitive to scrolling).

### Additional context

Sorry if this has already been discussed / decided upon and again, thanks on the amazing work on this tool!

**Comment by Wisser** [comment]
This topic has actually been discussed a few times. There is no unanimous opinion as to whether the mouse wheel should be used to zoom directly or only together with Control/Command.

I will make this configurable. In the attachment you can find a patch with a preview of this feature. To install it, unzip the file and replace the file "jailer.jar" in your installation with the one in the zip.

In the menu under "Settings" -> "Zoom with Mouse Wheel" you can set the behavior. Please give it a try.

Would you prefer the "Command" or the "Control" key on Mac?

**Comment by naxels** [comment]
Thank you!

This is absolutely perfect, works as expected.

I personally prefer the Command button on the Mac for important commands that you are likely to use often,
however don't know the Apple official guidelines on that.
Given that the zoom here is pretty relevant, I'd say let's go with Command unless you think it would collide with other keyboard shortcuts.

One thing I did find is when using the Magic Mouse 2 / Trackpad,
when 2 scrollbars become available in the window, 1 will dominate all scroll behaviour.
I tried it in another Java GUI app that works on Mac and there it didn't occur.

The other Java GUI app is Pentaho Data Integration, however don't know Java well enough to figure out why it works there.
Here is the repo in case you want to investigate:
https://github.com/pentaho/pentaho-kettle

Thanks again!
